step-templates/octopus-register-listening-target.json

```diff
@@ -1,135 +1,135 @@
-        "Label": "Port Number",
-        "HelpText": "The port the tentacle is listening on",
-        "DefaultValue": "10933",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "fa4bd89d-bb86-4d3f-87ff-17125cd88f24",
-        "Name": "RegisterListeningTarget.Roles.List",
-        "Label": "Role CSV List",
-        "HelpText": "Comma separated list of environments to assign to the machine in Octopus Deploy.",
-        "DefaultValue": "",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "0c80326d-cdc1-439d-be6b-fbab4da42cda",
-        "Name": "RegisterListeningTarget.Environment.List",
-        "Label": "Environment CSV List",
-        "HelpText": "Comma separated list of environments to assign to the the machine in Octopus Deploy.",
-        "DefaultValue": "",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "667a18b3-1694-4333-87a4-9be712b59122",
-        "Name": "RegisterListeningTarget.Tenant.List",
-        "Label": "Tenant CSV List",
-        "HelpText": "(Optional) If this is for a tenant, the a comma separated list of tenants to assign the machine to in Octopus Deploy",
-        "DefaultValue": "",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "db2de612-45bb-4e4a-ab95-64083dd80393",
-        "Name": "RegisterListeningTarget.Tenant.DeploymentType",
-        "Label": "Tenanted Deployments",
-        "HelpText": "Choose the kind of deployment where this deployment target should be included.",
-        "DefaultValue": "Untenanted",
-        "DisplaySettings": {
-          "Octopus.ControlType": "Select",
-          "Octopus.SelectOptions": "Untenanted|Exclude from tenanted deployments (default)\nTenanted|Include only in tenanted deployments\nTenantedOrUntenanted|Include in both tenanted and untenanted deployments"
-        }
-      },
-      {
-        "Id": "5ef91f37-b13b-413d-955c-872c7f274c7e",
-        "Name": "RegisterListeningTarget.MachinePolicy.IdOrName",
-        "Label": "Machine Policy Id Or Name",
-        "HelpText": "Enter in the name or the Id of the Machine Policy in Octopus Deploy for the AKS Cluster.",
-        "DefaultValue": "Default Machine Policy",
-        "DisplaySettings": {
-          "Octopus.ControlType": "SingleLineText"
-        }
-      },
-      {
-        "Id": "5b2e0993-7f96-4447-b47b-029c8f225688",
-        "Name": "RegisterListeningTarget.Overwrite.Existing",
-        "Label": "Overwrite Existing Registration",
-        "HelpText": "Indicates if the existing listening tentacle should be overwritten in Octopus.",
-        "DefaultValue": "False",
-        "DisplaySettings": {
-          "Octopus.ControlType": "Checkbox"
-        }
+    {
+      "Id": "25dd9d35-448e-4536-b622-d8d2fce8d4f4",
```

Comment:
This looks like a new ID, did you paste in the entire JSON body or just the script?